components/PaywallModal.vue

```diff
@@ -93,7 +92,6 @@
           <!-- Price Select -->
           <div class="flex flex-col w-full mt-12">
             <UButton
-              v-if="props.isDismissible"
```

Comment:
is this intentional?

Reply:
Since we can't control follow up actions from backdrop clicks, it's better to separate dismiss and close button handling.
[🚸 Split isDismissible logic](https://github.com/likecoin/3ook-com/pull/57/commits/d43015d46b4570d72d7f60b584f00c8242aa80ca)